plugin/evm/config/config.go

```diff
@@ -224,65 +158,6 @@ type Config struct {
 	HttpBodyLimit uint64 `json:"http-body-limit"`
 }
 
-// EthAPIs returns an array of strings representing the Eth APIs that should be enabled
-func (c Config) EthAPIs() []string {
-	return c.EnabledEthAPIs
-}
-
-func (c Config) EthBackendSettings() eth.Settings {
-	return eth.Settings{MaxBlocksPerRequest: c.MaxBlocksPerRequest}
-}
-
-func (c *Config) SetDefaults() {
-	c.EnabledEthAPIs = defaultEnabledAPIs
-	c.RPCGasCap = defaultRpcGasCap
-	c.RPCTxFeeCap = defaultRpcTxFeeCap
-	c.MetricsExpensiveEnabled = defaultMetricsExpensiveEnabled
-
-	c.TxPoolPriceLimit = legacypool.DefaultConfig.PriceLimit
-	c.TxPoolPriceBump = legacypool.DefaultConfig.PriceBump
-	c.TxPoolAccountSlots = legacypool.DefaultConfig.AccountSlots
-	c.TxPoolGlobalSlots = legacypool.DefaultConfig.GlobalSlots
-	c.TxPoolAccountQueue = legacypool.DefaultConfig.AccountQueue
-	c.TxPoolGlobalQueue = legacypool.DefaultConfig.GlobalQueue
-	c.TxPoolLifetime.Duration = legacypool.DefaultConfig.Lifetime
```

Comment:
A perhaps cleaner (or not?) way to do this would be to pass in these fields in a struct you define in the plugin/evm/config package i.e. `config.LegacyPoolDefaults` which the caller could set with fields from the legacypool package.